pkg/server/problem_ranges.go

```diff
@@ -289,7 +289,7 @@ const debugProblemRangesTemplate = `
         <DIV CLASS="row">
           {{- if $.LeaderNotLeaseholder}}
             {{- range $_, $r := $.LeaderNotLeaseholder}}
-              <a href="/debug/range?id={{$r}}">{{$r}}</a>
+              <a href="/debug/range?id={{$r}}">{{$r}}</a> &nbsp;
```

Comment:
So the browser is breaking at non-breaking spaces? Weird. Semantically, breaking whitespace is what you want here. I think if you want a breaking space that the template won't eat you can use `{{" "}}`.